shared/versions/stack/8.17.asciidoc

```diff
@@ -0,0 +1,75 @@
+:version:                8.17.0
+////
+bare_version never includes -alpha or -beta
+////
+:bare_version:           8.17.0
+:logstash_version:       8.17.0
+:elasticsearch_version:  8.17.0
+:kibana_version:         8.17.0
+:apm_server_version:     8.17.0
+// branch is 8.17 instead of 8.x for long-term linking purposes
+:branch:                 8.17
+:minor-version:          8.17
+:major-version:          8.x
+:prev-major-version:     7.x
+:prev-major-last:        7.17
+:major-version-only:     8
+:ecs_version:            8.17
```

Comment:
> Looks like there isn't a 8.17 branch yet in https://github.com/elastic/ecs

Reply:
There is a comment to not include 8.17 for ecs.